go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.4.0
 	github.com/mattn/go-sqlite3 v1.14.18
 	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/ginkgo/v2 v2.13.2
```

Comment:
we also talked about this offline - making this a direct dependency is good but I find the changed vendored files suprising..